src/main/java/org/snomed/snowstorm/rest/RelationshipController.java

```diff
@@ -83,9 +83,12 @@
 	private void expandSourceTypeAndDestination(String branch, List<Relationship> relationships, List<LanguageDialect> languageDialects) {
```

Comment:
I have changed the default constructor for Relationship to set the id to null instead of a blank String. Because of this, Elasticsearch was throwing an exception because of null fields; to fix this I have added a null check to RelationshipController.

Reply:
@dmcgihtsdo  Good catch. Can you just add a space in the if statement below before '(' and after ')' please?

Reply:
Fixed. My bad, I normally format before pushing.